### PR TITLE
Export API - Promote deleted team API from Beta to V1

### PR DESCRIPTION
**Pull request by AGLIU123**

### CODE REVIEW

**Comment by dodaromike** [comment]
@AGLIU123  Please resolve conflicts.

**Comment by AGLIU123** [comment]
Hi @dodaromike, thank you for the comment, I have resolved the conflict. :)

**Comment by AGLIU123** [comment]
Hi @dodaromike , here are the [API.md](https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/12241?path=/reviews/15540-export-api-promote-deleted-team-api-to-v1/api.md&discussionId=148710&_a=files) and [CSDL files change](https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/7602177?path=/Workloads/Microsoft.Teams.GraphSvc/override/schema-Fairfax-v1.0.csdl&_a=files).

**Comment by dodaromike** [comment]
@AGLIU123 API Doctor doesn't like the request example in getAllTeams. It doesn't seem to like the parentheses there for the placeholder. Can you add real examples instead of placeholders? It looks like the beta version of that topic has a real example and it didn't fail.

**Comment by AGLIU123** [comment]
Hi @dodaromike , could you help to add more details about "request example" or help to add the comment to changed file please? Like which file is it for (getAllMessage or deletedTeam api) and is it for request url? Thank you in advance.

**Comment by dodaromike** [comment]
@AGLIU123  It could be the beta equivalent of this file:  https://learn.microsoft.com/en-us/graph/teams-list-all-teams

**Comment by dodaromike** [comment]
This is the problem example: line 66 in getAllMessages.md

## Examples

### Request
The following is an example of a request.

**Comment by AGLIU123** [comment]
Thank you for the clarification @dodaromike , I have updated the request example.

File: api-reference/v1.0/resources/deletedteam.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+A deleted team in Microsoft Teams is a collection of [channel](channel.md) objects. A channel represents a topic, and therefore a logical isolation of discussion, within a deleted team.
+
+Every deleted team is associated with a [Microsoft 365 group](../resources/group.md). For more information about working with groups and members in teams, see [Use the Microsoft Graph REST API to work with Microsoft Teams](teams-api-overview.md).
+
+Inherits from [entity](../resources/entity.md).
+
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[getAllMessages](../api/deletedteam-getallmessages.md)|[chatMessage](../resources/chatmessage.md) collection|Get all messages in the deleted team.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/deletedteam.md:23]: The methods table here should also list the List deletedTeams operation. Please make this update to the deletedTeam resource in the beta folder as well. Please also remove "(preview)" from the TOC entry for Deleted team in the TOC.yml file in the beta folder. Now that this is going GA, we should not indicate that the API is in preview.

File: api-reference/v1.0/resources/deletedteam.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|The ID of a deleted team. Inherited from [entity](../resources/entity.md).|
+
+## Relationships
+|Relationship|Type|Description|
+|:---|:---|:---|
+|channels|[channel](../resources/channel.md) collection|The channels those are either shared with this deleted team or created in this deleted team.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/deletedteam.md:33]: For the description, I think what you mean here is "The channels that are either...". Please fix.

File: api-reference/v1.0/resources/deletedteam.md (hunk 1 above)
Comment (AGLIU123) [reply api-reference/v1.0/resources/deletedteam.md:23]: Thank you for the comment, I just added the List deletedTeams operation in both v1 and beta doc and removed "(preview)" from the TOC.

**Review by dodaromike — APPROVED** [review]

**Pull request merged**